AirCasting/SessionViews/SessionHeaderView.swift

```diff
@@ -13,12 +13,23 @@ struct SessionHeaderView: View {
     let action: () -> Void
     let isExpandButtonNeeded: Bool
     @ObservedObject var session: Session
-        
+    @EnvironmentObject private var microphoneManager: MicrophoneManager
+    
     var body: some View {
         VStack(alignment: .leading, spacing: 13){
             dateAndTime
             nameLabelAndExpandButton
-            measurements
+            if (session.deviceType == DeviceType.MIC.rawValue) {
```

Comment:
should we reference rawValue in comparison? Maybe it would be better to use an enum case in `session`